src/box/errcode.h

```diff
@@ -99,10 +99,12 @@ struct errcode_record {
 	_(ER_TEST_TYPE_STRING, 10008,		"Test error", "field", STRING) \
 	_(ER_TEST_TYPE_MSGPACK, 10009,		"Test error", "field", MSGPACK) \
 	_(ER_TEST_TYPE_TUPLE, 10010,		"Test error", "field", TUPLE) \
-	_(ER_TEST_2_ARGS, 10011,		"Test error", "f1", INT, "f2", INT) \
-	_(ER_TEST_3_ARGS, 10012,		"Test error", "f1", INT, "f2", INT, "f3", INT) \
-	_(ER_TEST_4_ARGS, 10013,		"Test error", "f1", INT, "f2", INT, "f3", INT, "f4", INT) \
-	_(ER_TEST_5_ARGS, 10014,		"Test error", "f1", INT, "f2", INT, "f3", INT, "f4", INT, "f5", INT) \
+	_(ER_TEST_2_ARGS, 10030,		"Test error", "f1", INT, "f2", INT) \
+	_(ER_TEST_3_ARGS, 10031,		"Test error", "f1", INT, "f2", INT, "f3", INT) \
+	_(ER_TEST_4_ARGS, 10032,		"Test error", "f1", INT, "f2", INT, "f3", INT, "f4", INT) \
+	_(ER_TEST_5_ARGS, 10033,		"Test error", "f1", INT, "f2", INT, "f3", INT, "f4", INT, "f5", INT) \
+	_(ER_TEST_FORMAT_MSG, 10050,		"Test error %i %s", "f1", INT, "f2", STRING) \
+	_(ER_TEST_FORMAT_MSG_LESS, 10051,	"Test error %i %s", "f1", INT, "f2", STRING, "f3", INT) \
```

Comment:
Nit: "fewer" not "less"

Reply:
Fixed.

<details>
<summary>diff</summary>

```diff
diff --git a/src/box/errcode.h b/src/box/errcode.h
index 8cd39cc6e..b25fa1cc0 100644
--- a/src/box/errcode.h
+++ b/src/box/errcode.h
@@ -104,7 +104,7 @@ struct errcode_record {
 	_(ER_TEST_4_ARGS, 10102,		"Test error", "f1", INT, "f2", INT, "f3", INT, "f4", INT) \
 	_(ER_TEST_5_ARGS, 10103,		"Test error", "f1", INT, "f2", INT, "f3", INT, "f4", INT, "f5", INT) \
 	_(ER_TEST_FORMAT_MSG, 10200,		"Test error %i %s", "f1", INT, "f2", STRING) \
-	_(ER_TEST_FORMAT_MSG_LESS, 10201,	"Test error %i %s", "f1", INT, "f2", STRING, "f3", INT) \
+	_(ER_TEST_FORMAT_MSG_FEWER, 10201,	"Test error %i %s", "f1", INT, "f2", STRING, "f3", INT) \
 
 #else
 #define TEST_ERROR_CODES(_)
diff --git a/test/box-luatest/error_subsystem_improvements_test.lua b/test/box-luatest/error_subsystem_improvements_test.lua
index abd96c774..448db791e 100644
--- a/test/box-luatest/error_subsystem_improvements_test.lua
+++ b/test/box-luatest/error_subsystem_improvements_test.lua
@@ -279,7 +279,7 @@ g.test_client_error_creation = function()
 
     -- Test number of arguments of format string may be less
     -- then number of payload arguments.
-    local e = box.error.new(box.error.TEST_FORMAT_MSG_LESS, 1, 'five', 3)
+    local e = box.error.new(box.error.TEST_FORMAT_MSG_FEWER, 1, 'five', 3)
     t.assert_equals(e.message, 'Test error 1 five')
     t.assert_equals(e.f1, 1)
     t.assert_equals(e.f2, 'five')
diff --git a/test/unit/error.c b/test/unit/error.c
index 4b8ce893e..3e9cd4d5d 100644
--- a/test/unit/error.c
+++ b/test/unit/error.c
@@ -779,7 +779,7 @@ test_client_error_creation(void)
 	 * Test number of arguments of format string may be less
 	 * then number of payload arguments.
 	 */
-	diag_set(ClientError, ER_TEST_FORMAT_MSG_LESS, 1, "seven", 3);
+	diag_set(ClientError, ER_TEST_FORMAT_MSG_FEWER, 1, "seven", 3);
 	e = diag_last_error(diag_get());
 	ok(error_get_int(e, "f1", &i) && i == 1);
 	s = error_get_str(e, "f2");
```

</details>